### ISSUE TITLE
Create new instance Eleoquent model without providing attributes

### ISSUE THREAD

**Issue by reno1979** [laravel/framework#15974]
- Laravel Version: 5.3.19 (thank you for the multiple connection fix!!)
- PHP Version: 7
- Database Driver & Version: Mysql (?) 

- extension (translate)[https://github.com/dimsav/laravel-translatable]

### Description:
Creating default object from empty value

### Steps To Reproduce:
Create a model plus table with only an id field (incrementing) 
(the name and description columns are translated values that exist in another table) 

``` 
$fillable = [];
$guarded = [];
``` 

calling create() without providing any attributes will cause the Creating default object from empty value message.

#### Update
I have solved it by using a dummy column (inside migration and inside the model), getting a dummy default value. 
```
$attributes = ['dummy' => ""]; 
$fillable = ['dummy'];
$guarded = [];
```

**Comment by themsaid** [comment]
I don't seem to be able to replicate the issue:

```
class Test extends Authenticatable
{
    protected $table = 'tests';

    public $timestamps = false;
}
```

Calling `\App\Test::create();` creates a record in the DB as normal.

**Comment by reno1979** [comment]
@themsaid 

You are absolutely right, I removed everything from my model and now the create method works without requiring attributes.

I'll try to see if it's caused by the translatable plugin. 

Calling the create method when setting a different connection is still giving me problems though. 

``` 
$featureGroup = \App\FeatureGroup::on('mysql2')->getModel()->create();
``` 

I tried using getModel, because after calling on() I get a Builder instance.
